spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -267,3 +404,51 @@ definitions:
             desc: |
               Phase biases corrections for a
               satellite being tracked.
+
+
+ - MSG_SSR_STEC_CORRECTION:
+    id: 0x05EB
+    short_desc: Slant Total Electon Content
+    desc: |
+      The STEC per space vehicle, given as polynomial approximation for
+      a given grid.  This should be combined with SSR-GriddedCorrection
+      message to get the state space representation of the atmospheric
+      delay.
+    fields:
+        - header:
+            type: STECHeader
+            desc: Header of a STEC message
+        - stec_sat_list:
+            type: array
+            fill: STECSatElement
+            desc: Array of STEC information for each space vehicle
```

Comment:
The STECHeader defines a `index/total_messages` scheme for create a sequence of MSG_SSR_STEC_CORRECTION messages, in that case does this need to be an array?

Reply:
Or, I guess the idea is to put as many `STECSatElement` into one message as are possible?

Reply:
yup!